index.js

```diff
@@ -3983,6 +3983,47 @@
   S.splitOn =
   def('splitOn', {}, [$.String, $.String, $.Array($.String)], splitOn);
 
+
```

Comment:
Nit: Please remove this line so that there's just a single empty line between functions.